### PR TITLE
Route MCI/RSI based on empty/zero Total Retail sales

### PR DESCRIPTION
**Pull request by dwyeradam**
Introduce the concept of a "default" value.
This default value is stored in answer store on POST if no value is given for the answer.
Reduce duplication on conditions in rules.py

### CODE REVIEW

File: app/questionnaire/rules.py [hunk 1]
```diff
@@ -16,30 +16,11 @@ def evaluate_rule(when, answer_value):
...
-        result = True
-    elif condition == 'not set':
-        result = answer_value is None
-    elif condition == 'greater than' and isinstance(answer_value, int) and answer_value > match_value:
-        result = True
-    elif condition == 'less than' and isinstance(answer_value, int) and answer_value < match_value:
-        result = True
-
-    return result
+    return evaluate_condition(condition, match_value, answer_value)
```
Comment (jonnyshaw89) [inline app/questionnaire/rules.py:20]: 👍 A Positive change

File: scripts/test_schemas.sh [hunk 2]
```diff
@@ -32,11 +32,17 @@ done
 
 exit=0
 
-echo "---Testing Schemas---"
+if [ $# -eq 0 ]; then
+    file_path="./data/en"
```
Comment (jonnyshaw89) [inline scripts/test_schemas.sh:36]: 👍 Helpful

File: app/questionnaire/rules.py (hunk 1 above)
Comment (dwyeradam) [reply app/questionnaire/rules.py:20]: I should have done it with the date routing stuff really so better late than never.

File: scripts/test_schemas.sh [hunk 3]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
-docker pull onsdigital/eq-schema-validator
-validator="$(docker run -d -p 5001:5000 onsdigital/eq-schema-validator)"
+docker pull onsdigital/eq-schema-validator:add-deafult-for-numeric-answers
```
Comment (nealedj) [inline scripts/test_schemas.sh:3]: Standard friendly reminder to revert this before merge.

**Review by nealedj — APPROVED** [review]

**Comment by jonnyshaw89** [comment]
@nealedj He says as he merges it????

**Comment by nealedj** [comment]
Apparently my memory times out after a couple of minutes on Friday afternoons...

**Pull request merged**